Add already fixed bug to 1.65.0 release notes (#4323)
FusionAuth/fusionauth-site: astro/src/content/releases/v1-62-0.mdx

In the Admin UI, the Twilio or Generic Messenger **send test message** button always produced an unexpected error.
</Item>

<Item category="known-issue" resolvedIn="1.65.0" issue="3436">

> When viewing keys in Key Master without an algorithm, you will get a Freemarker error at the top of the view menu.

Reviewer: ```suggestion
When viewing keys without an algorithm, Key Master displays a [FreeMarker](https://freemarker.apache.org/) error at the top of the view menu.
```
Minor wording improvements + link to freemarker + capitalize M to follow apache's example
